src/status_im/contexts/wallet/events.cljs

```diff
@@ -369,7 +384,7 @@
 
 (rf/reg-event-fx :wallet/reload
  (fn [_]
-   {:fx [[:dispatch-n [[:wallet/get-wallet-token]]]]}))
+   {:fx [[:dispatch-n [[:wallet/get-wallet-token-for-all-accounts]]]]}))
```

Comment:
We can remove the redundant `:dispatch-n` since we are already  touching this event :eyes:

Reply:
Sure 👍